support/openharmony/entry/src/main/ets/entryability/EntryAbility.ets

```diff
@@ -5,8 +5,30 @@ import Want from '@ohos.app.ability.Want';
 import window from '@ohos.window';
 
 export default class EntryAbility extends UIAbility {
+  init_params: LocalStorage = new LocalStorage();
   onCreate(want: Want, launchParam: AbilityConstant.LaunchParam) {
-    hilog.info(0x0000, 'testTag', '%{public}s', 'Ability onCreate');
+    let uri: string = want.uri || "https://servo.org"
+    let params: string[] = []
+    if (typeof want.parameters !== "undefined") {
+      Object.entries(want.parameters).forEach((entry) => {
+        let key = entry[0]
+        // Skip some default parameters, since servo is not interested in those
+        if (key.startsWith("ohos.")
+          || key.startsWith("component.")
+          || key === "isCallBySCB"
+          || key === "moduleName"
+        ) {
+          return
+        }
+        let value = entry[1]
+        params.push("--" + key + "=" + value.toString())
+        hilog.info(0x0000, 'Servo Entryability', 'Parameters: %s=%s', key, JSON.stringify(value))
```

Comment:
Is the `{public}` modifier not needed here in `%s=%s`?

Reply:
Yeah, it should be added. But I removed the log statement instead, since it is overly verbose.